requirements.txt

```diff
@@ -33,12 +33,9 @@ lockfile==0.12.2
 logshipper@ git+https://github.com/StackStorm/logshipper.git@stackstorm_patched ; platform_system=="Linux"
 mock==5.1.0
 mongoengine==0.23.1
-networkx==2.8.8
-nose
-nose-parallel==0.4.0
-nose-timer==1.0.1
+networkx==3.1
 orjson==3.10.7
-orquesta@ git+https://github.com/StackStorm/orquesta.git@v1.6.0
+orquesta@ git+https://github.com/nzlosh/orquesta.git@st2v39_compat
```

Comment:
This needs to be merged into orquesta master

Reply:
The MR is open https://github.com/StackStorm/orquesta/pull/266/files